challenge.rb

```diff
@@ -44,8 +48,20 @@ def okay_to_drink? age, drinking_age
 # If the AC is non-functional and it's NOT too hot, `ac_monitor` should return
 #   "Fix the AC whenever you have the chance. It's cool."
 
-def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+def ac_monitor(current_temp, ac_working, desired_temp)
+  if ac_working
+    if current_temp > desired_temp
+      return 'Turn on the AC.'
+    else
+      return 'Just right!'
+    end
+  else
+    if current_temp > desired_temp
+      return 'Fix the AC now! It\'s hot!'
+    else
+      return 'Fix the AC whenever you have the chance. It\'s cool.'
+    end
+  end
```

Comment:
Yep! In fact, because Ruby implicitly returns the last evaluated expression, you could even have left off `return` and just said 
```
if current_temp > desired_temp
  'Turn on the AC.'
else
...
```
etc